Documentation/ProjectFile/material/fracture_properties/permeability_model/ConstantHydraulicAperture/c_ConstantHydraulicAperture.md

```diff
@@ -0,0 +1,4 @@
+The permeability model is similar to the cubic law, but uses the initial
+aperture \f$b_0\f$ for the hydraulic part, s.t. for the mechanical aperture
+\f$b_m\f$ the resulting permeability is
+\f[ K(b_m; b_0) = b_0\cdot b_m^2/12.\f]
```

Comment:
as i already commented in previous PR, this formulation is not for permeability.